src/tableaus.jl

```diff
@@ -1,37 +1,153 @@
 """
-$(TYPEDEF)
+  $(TYPEDEF)
```

Comment:
I think this should just be
```suggestion
$(TYPEDEF)
```